build/sources/ceph-builders.upload

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+export OS_AUTH_URL="$OVH_AUTH_URL"
```

Comment:
That's a perfectly good remark. Even more that script should not be here. That's to specific for my usage. I'll move it out of edeploy.